invenio_base/wsgi.py

```diff
@@ -53,11 +55,51 @@ def wsgi_proxyfix(factory=None):
        otherwise you application is susceptible to malicious attacks.
 
     .. versionadded:: 1.0.0
+
+    .. note::
+
+       You must set ``APP_PROXY_CONFIG`` with the correct values if your
+       application is running behind a proxy, otherwise your application is
+       susceptible to malicious attacks because to trust values that came from
+       the client rather than a proxy is a security vulnerability. The
+       configuration is a dictionary which looks like this:
+
+       .. code-block:: python
+
+          PROXYFIX_CONFIG = {
+              x_for: 1,  # X-Forwarded-For header set by proxy 1 will be read
```

Comment:
I am not sure it is easy to understand what is `proxy 1`, or maybe it is only me :)
```
X-Forwarded-For header: Invenio will read only the first value and ignore any other appended values
```